plugin/src/main/kotlin/com/nishtahir/CargoBuildTask.kt

```diff
@@ -42,6 +44,10 @@ open class CargoBuildTask : DefaultTask() {
                 ?: targetDirectory
                 ?: "${module!!}/target"
 
+            if (Files.exists(Paths.get(targetDirectory))) {
+                throw GradleException("Cargo target directory (`${targetDirectory}`) is not exists. If you're using Cargo Workspace, you need to specify the target directory in `build.gradle` or `local.properties` (see https://github.com/mozilla/rust-android-gradle#targetdirectory).")
```

Comment:
nit: s/is not exists/does not exist/.  And s/Cargo Workspace/a Cargo workspace/.

Reply:
Updated.